hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientCacheNearCachePreloaderTest.java

```diff
@@ -181,6 +181,10 @@ protected ClientConfig getClientConfig() {
                 .setNearCacheAdapter(new ICacheDataStructureAdapter<K, V>(clientCache))
                 .setNearCache(nearCache)
                 .setNearCacheManager(nearCacheManager)
-                .setCacheManager(cacheManager);
+                .setCacheManager(cacheManager)
+                // FIXME: the JCache doesn't send invalidation on CREATED entries, so this will crash some tests
```

Comment:
I just wanted to add it here in the same way like in the Near Cache basic test, so it's not easily forgotten if we should add those invalidations for JCache as well (see `ClientCacheNearCacheBasicTest`).